frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx

```diff
@@ -18,9 +18,18 @@ const isTableLinkDisabled = (table: LineageItem) => {
   const config = AppConfig.tableLineage;
```

Comment:
Is there a function to retrieve this configuration instead of using it directly?